### DISCUSSION TITLE
为雷鸟寻找新的东家 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#96] [Announcements]
# 为雷鸟寻找新的东家 - 「开源之道」

https://opensourceway.community/posts/opensource/finding_a_home_for_thunderbird/

**Comment by ZoomQuiet** [comment]
> 是也乎,(￣▽￣)

类似的, 目标是打破信息孤岛的软件有很多, 都没什么好下场, 
毕竟无论多优秀, 改变人类固有懒散天性的东西, 总是会慢消失...

最后的译按, 比现在的要尖锐很多...

不过, 优秀的代码和项目, 为什么会被放弃?

原因, 可能就是, 世界已经不需要了...
